tests/console/oprofile.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple tests for oprofile.
+# - profile example application (fio to create some load)
+# - check report
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # Install fio as load generator and oprofile
+    zypper_call "in fio oprofile";
+
+    # Start a system wide profiling
+    type_string "operf --system-wide &";
```

Comment:
```suggestion
    script_run "(operf --system-wide &)";
```

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10656#discussion_r455002094 
`script_run` did not work for me for background jobs and I found other tests using `type_string` as workaround. 
If you say it is supposed to work (maybe a parenthesis issue) I will try one more time

Reply:
Yes, try with parenthesis 😄

Reply:
with type_string you have to press return `\n`